commands/lxc-to-lxd

```diff
@@ -18,8 +18,8 @@ except ImportError:
 # is strictly only true for 'lxc.aa_profile'.
 keys_to_check = [
     'lxc.pts',
-    # 'lxc.tty',
-    # 'lxc.devttydir',
+    'lxc.tty',
```

Comment:
You're including them in the list to ignore them - is this a whitelist? And so they're ignored because they're allowed in the config but we don't do anything with them, is that right?